scripts/build/windows.bat

```diff
@@ -212,10 +212,7 @@ call %electron_packager% . %application_name%^
  --out=%output_build_directory%
 
 set package_output=%output_build_directory%\%package_name%
-
-if not "%arch%"=="%electron_arch%" (
-  move %output_build_directory%\Etcher-win32-%electron_arch% %package_output%
-)
```

Comment:
Totally right. The line should be changed to what @lurch suggested. I did try a 32-bit build, however I guess I had stuff on my `etcher-release` before running the command, which made the issue slip undetected. Sorry for that!